web/annotation_editor_layer_builder.css

```diff
@@ -331,4 +361,182 @@
       }
     }
   }
+
+  &
+    :is(
+      [data-main-rotation="0"] [data-editor-rotation="90"],
+      [data-main-rotation="90"] [data-editor-rotation="0"],
+      [data-main-rotation="180"] [data-editor-rotation="270"],
+      [data-main-rotation="270"] [data-editor-rotation="180"],
+
+    ) {
+    & .altText {
+      rotate: 270deg;
+
+      &[dir="ltr"] {
```

Comment:
To maintain consistency with other CSS files, could we perhaps use `:dir(ltr)` respectively `:dir(rtl)` instead in this patch?